tasks/protractor_coverage.js

```diff
@@ -103,11 +103,11 @@ module.exports = function(grunt) {
     coverageDir = coverageDir.replace(/\\/g,'/');
     var noInject = opts.noInject;
     if (!noInject) {
-      var saveCoverageTemplate = grunt.file.expand([ opts.saveCoverageTemplate, "node_modules/grunt-protractor-coverage/resources/saveCoverage.tmpl", process.cwd()+'/**/resources/saveCoverage.tmpl']).shift();
```

Comment:
You are right the name should not contain cucumber. noInject is used for cucumber, this should definitely not be called that. I will rename it.